### PR TITLE
Add photon.pb.h to cpp headers zip

### PR DESCRIPTION
**Pull request by mcm001**
Previously, the header was bundled with the sources zip.

### CODE REVIEW

**Comment by mcm001** [comment]
Blocked until we can figure out how to publish photonverison.cpp

File: photon-targeting/src/main/native/include/photon/dataflow/structures/Packet.h [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #include <bit>
 #include <string>
 #include <vector>
+#include <cstring>
```
Comment (srimanachanta) [inline photon-targeting/src/main/native/include/photon/dataflow/structures/Packet.h:24]: was this missing before or just for redundancy?

**Review by srimanachanta — APPROVED** [review]

**Comment by srimanachanta** [comment]
Needs formatting fixes

File: photon-targeting/src/main/native/include/photon/dataflow/structures/Packet.h (hunk 1 above)
Comment (mcm001) [reply photon-targeting/src/main/native/include/photon/dataflow/structures/Packet.h:24]: Missing

**Pull request merged**